### DISCUSSION TITLE
Github SaaS Possible Feature Request: Is there a way to force all repositories in an organisation to have a minimum of 1 reviewer

### DISCUSSION THREAD

**Discussion by efernandes-dev-ops** [community/community#21602] [New to GitHub]
<p>Hi we are currently using github saas within our organisation. We have multiple organisations setup and would like to have a policy applied across all repository’s within the org to have a minimum of 1 reviewer. Is this possible at all?</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave:  Hi!</p>
<p>It isn’t possible right now I’m afraid. That said, you’re not alone in wanting this, and I’ve added your request to the internal tracking issue.</p>
<p>I’ll do my best to keep you updated if there’s something to report.</p>
